src/RPCServer.ts

```diff
@@ -20,11 +20,11 @@ import { CreateDestroy, ready } from '@matrixai/async-init/dist/CreateDestroy';
 import Logger from '@matrixai/logger';
 import { PromiseCancellable } from '@matrixai/async-cancellable';
 import { Timer } from '@matrixai/timer';
-import { RawHandler } from '@/handlers/rawHandler';
-import { DuplexHandler } from '@/handlers/duplexHandler';
-import { ServerHandler } from '@/handlers/serverHandler';
-import { ClientHandler } from '@/handlers/clientHandler';
-import { UnaryHandler } from '@/handlers/unaryHandler';
+import { RawHandler } from '@/handlers/RawHandler';
+import { DuplexHandler } from '@/handlers/DuplexHandler';
+import { ServerHandler } from '@/handlers/ServerHandler';
+import { ClientHandler } from '@/handlers/ClientHandler';
+import { UnaryHandler } from '@/handlers/UnaryHandler';
```

Comment:
FYI, you can only use `@` in the import path in tests. In the source files you must use relative path for imports. It will build, tests will run. But running outside of tests will have runtime errors..

Reply:
Gotcha, thanks